modules/statics/src/account.ts

```diff
@@ -331,22 +331,6 @@ export class AdaCoin extends AccountCoinToken {
   }
 }
 
-/**
- * The Trx network supports tokens
- * Trx tokens are identified by their smart contract address
- *
- */
-export class TrxCoin extends AccountCoinToken {
```

Comment:
Assuming we're deleting all of these because we're just using TronERC20Token() instead?

Reply:
yes, i overlooked that there was a TronERC20Token() class before (it's used for the wrapped coins)